### ISSUE TITLE
Doesn't do map missions

### ISSUE THREAD

**Issue by AphroBytes** [Dandey/Firestone-bad-bot#1]
Hi, as per the title using the latest script it doesn't seem to start map missions, it just looks at the map and then goes to the next function.

Any ideas why?

Using the resolution specified in the readme - Windows 10,  Steam version of the game.

**Comment by Dandey** [comment]
Do you have the game in another language? The script starts the map missions by looking for the idle squads message at the bottom of the screen.

**Comment by Dandey** [comment]
Will double check later today when i'll get my next map reset.

**Comment by Dandey** [comment]
Something about the window size, actually checked what my game size is as I was unsure and turn out it ends up on 1920x1074, so if you stay on 1920x1080 it might go bad on coords/ image searches.

**Comment by Dandey** [comment]
Hmm, it loops somewhere, please add me on discord for a bit of an easier debugging. discord.com/users/187582178606055424

**Comment by Dandey** [comment]
Any other issue appearing for the rest of script? Last someone mentioned issue on map, return to main menu after gem confirmation didn't worked as well, and after the resolution "fix"(not sure how to properly make the script recognise a resolution to work only on that) that worked as well along with the map function.

**Comment by Dandey** [comment]
Can you give it a try with this version? Not sure it would help much but tried something, there is a chance that the loop is fixed, but most likely it will run into map nodes issue https://github.com/Dandey/Firestone-bad-bot/releases/tag/Map-beta

**Comment by Dandey** [comment]
It's most likely something related to resolution, that was also the reason it was ending on loop on closing, I'm surprised how the gem confirmation does not get skip instead of just going along with the normal flow of alchemist/ firestone instead of just closing, and not sure if i can somehow make the nodes scale along with the game
As i matter of fact this is how i make the game "ready" for the script as when i'm not scripting i'm keeping it small

https://github.com/Dandey/Firestone-bad-bot/assets/17856136/f4802308-7e0a-492d-a48e-0546cbf9488f

**Comment by Dandey** [comment]
If this doesn't work I'm out of ideas, game will move alone on your main display( will have to look if I can make it remain on the display where the game was before starting the script)
https://github.com/Dandey/Firestone-bad-bot/releases/tag/Map-beta

**Comment by Dandey** [comment]
Sorry for late reply, meant to get back earlier but wasn't able, I try every change I make for at least an hour so it could do twice a cycle of everything, sometime I overlook stuff so I come back with fixes

**Comment by Dandey** [comment]
I think I might have a lead regarding this issue, someone that mentioned the same flow on discord was nice enough to help me a bit, and noticed that for some reasons sometimes the map screen have different coordinates(instead of 800-600 it was something like 900-1100), will have to look further into it to learn why it goes like that. May i ask if you have your windows in another language than english?

**Comment by Dandey** [comment]
Not sure what could go wrong, only thing that I found that changes coords is the scale in windows(as I have made them on 100%) , not sure how it scales in case of monitor with higher resolution as I don't have to test it out
![image](https://github.com/Dandey/Firestone-bad-bot/assets/17856136/eae14bb8-ec85-4eef-9d52-36286f434d66)

**Comment by Dandey** [comment]
> > Multiple monitors setup? And if yes they are same resolution?
> 
> Nope, single monitor in use.
> 
> I have other monitors stored away that I could try; however, I don't think that would make a difference?

Yeah, probably wouldn't make any difference, will have to look more into it but no idea if i can figure it out

**Comment by Vikijaka** [comment]
Hey there, 

i just tried to record my "map loop/script not doing map mission" and this just happened... whatsoever.
I dont have a 1080p Screen - its happening via Steam and Epic Version - Windowssize: 1936x1056 
recorded via Win10/11 Snippingtool (Win+Shift+S)

Any chance/idea to fix this? Im quite sure i do the have the wrong window size due to having "that bar" attached to the games window (Name... minimize, maximize, close)
https://streamable.com/4zxthy 

Sorry record was too big for github and too long for imgur

**Comment by Dandey** [comment]
> Hey there,
> 
> i just tried to record my "map loop/script not doing map mission" and this just happened... whatsoever. I dont have a 1080p Screen - its happening via Steam and Epic Version - Windowssize: 1936x1056 recorded via Win10/11 Snippingtool (Win+Shift+S)
> 
> Any chance/idea to fix this? Im quite sure i do the have the wrong window size due to having "that bar" attached to the games window (Name... minimize, maximize, close) https://streamable.com/4zxthy
> 
> Sorry record was too big for github and too long for imgur

if the game is another resolution that the given one it would fail on some functions that have pixel/ image recognition and some functions like map that would need given coordinates to work
